<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidget</class>
 <widget class="QWidget" name="SettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>214</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="cancelPushButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="applyPushButton">
      <property name="text">
       <string>Apply</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>18</y>
     <width>391</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Theme :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="DayModeRadioButton">
        <property name="text">
         <string>Daytime mode</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="NightModeRadioButton">
        <property name="text">
         <string>Night mode</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>48</y>
     <width>391</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>CPU warning value:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="CPUWariningComboBox">
        <item>
         <property name="text">
          <string>5%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>20%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>25%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>40%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>50%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>70%</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>80%</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Mem warning value:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="MemWarningComboBox">
        <item>
         <property name="text">
          <string>500MB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>800MB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000MB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1500MB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2000MB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3000MB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4000MB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5000MB</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>8000MB</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Log Level :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="LogLevelComboBox">
        <item>
         <property name="text">
          <string>trace</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>debug</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>info</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>warn</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>err</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>critical</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>off</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Due date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="DueDateComboBox">
        <item>
         <property name="text">
          <string>500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>800</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1200</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>1500</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5000</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>10000</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>ms</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>(needs to restart to take effect)</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
